### ISSUE TITLE
Discipline Plate V2?

### ISSUE THREAD

**Issue by synwt** [coseyfannitutti/discipline#71]
Can I use the plate https://github.com/coseyfannitutti/discipline/blob/master/gerber/plate/discipline-plate.zip for Discipline PCB V2?

**Comment by NiciTheNici** [comment]
Yes you can, if you look at commit b40bc8f it shows that the plate has been updated to the V2 version
